### PR TITLE
[MOS-183] Fix RT1051 debug build

### PR DESCRIPTION
**Pull request by Lefucjusz**
Fix of multiple issues that resulted
in debug build failing.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp [hunk 1]
```diff
@@ -25,7 +25,7 @@ namespace gui::status_bar
         constexpr auto signal3_roaming = "gsm_3_roaming";
         constexpr auto signal4_roaming = "gsm_4_roaming";
 
-        using SignalMap = std::unordered_map<Store::RssiBar, std::string>;
+        using SignalMap = std::map<Store::RssiBar, std::string>;
```
Comment (dawidwojtas-mudita) [inline module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp:28]: What is the reason for changing `unordered_map` over `map`?

File: module-services/service-desktop/include/service-desktop/ServiceDesktop.hpp [hunk 2]
```diff
@@ -17,33 +17,42 @@
...
+    inline constexpr auto irqQueueName     = "irqQueueBuffer";
+    inline constexpr auto irqQueueItemSize = sizeof(bsp::USBDeviceStatus);
+    inline constexpr auto irqQueueLength   = 1;
+
+    inline constexpr auto signallingQueueName     = "signallingQueueBuffer";
+    inline constexpr auto signallingQueueItemSize = sizeof(WorkerDesktop::Signal);
+    inline constexpr auto signallingQueueLength   = 1;
+
+    inline constexpr auto connectionActiveTimerName    = "connectionActiveTimer";
+    inline constexpr auto connectionActiveTimerDelayMs = std::chrono::milliseconds{1000 * 20};
```
Comment (dawidwojtas-mudita) [inline module-services/service-desktop/include/service-desktop/ServiceDesktop.hpp:49]: Consider using `std::chrono_literals` and then use `2000ms` :)

File: module-sys/Service/include/Service/Worker.hpp [hunk 3]
```diff
@@ -48,8 +47,8 @@ namespace sys
 
     struct WorkerCommand
     {
-        uint32_t command = 0;
-        uint32_t *data   = NULL;
+        std::uint32_t command = 0;
+        std::uint32_t *data   = NULL;
```
Comment (dawidwojtas-mudita) [inline module-sys/Service/include/Service/Worker.hpp:51]: Use `nullptr` here.

**Review by dawidwojtas-mudita — APPROVED** [review]

**Review by lukasmas — APPROVED** [review]

File: module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp (hunk 1 above)
Comment (Lefucjusz) [reply module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp:28]: There was some very strange error during linking stage regarding `ceill` method (multiple symbol definition), by analyzing map file I found out that the only module that uses this function is somewhere in `SignalStrengthBar.cpp` and it turned out to be used in `unordered_map` container. It's not a proper fix, I'm aware of it, but I've run out of other ideas to resolve this issue. This map is rather small, so this change should not affect the performance noticeably.

File: module-sys/Service/include/Service/Worker.hpp (hunk 3 above)
Comment (Lefucjusz) [reply module-sys/Service/include/Service/Worker.hpp]: Fixed!

File: module-services/service-desktop/include/service-desktop/ServiceDesktop.hpp (hunk 2 above)
Comment (Lefucjusz) [reply module-services/service-desktop/include/service-desktop/ServiceDesktop.hpp]: Done!

File: module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp (hunk 1 above)
Comment (dawidwojtas-mudita) [reply module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp:28]: Ok, that's strange. Thanks for the investigation and explanation.

**Pull request merged**